articles/integrations/azure-api-management/configure-azure.md

```diff
@@ -0,0 +1,172 @@
+| Specification format | the API specification format. Use `Swagger`. |
+| New/Existing API | set this to **New** |
+| Web API URL suffix | the value appended to the base URL of your API management service that uniquely identifies the API you're currently creating, such as `calc` |
+| Web API URL scheme | the protocol used to access your API (for this example, set this to `HTTPs`) |
+| Products | Add this API to the `Starter` product. This is a basic, getting-started-with-Azure container that holds your sample products and applies entry-level rate limits to your calls. |
+
+![](/media/articles/integrations/azure-api-mgmt/azure/import-api-config.png)
+
+When done, click **Save** to import your API. You'll be redirected to the summary page for your API when it's fully imported.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/basic-calc-api.png)
+
+### Configure Your OAuth 2.0 Authorization Server
+
+To use Auth0 to secure your API, you'll need to register Auth0 as an OAuth 2.0 Authorization Server. You can do so using the Azure Publisher Portal.
+
+Navigate to **Security** > **OAuth 2.0**.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/oauth2-servers.png)
+
+Click on **Add Authorization Server**. You'll see the configuration screen that lets you provide details about your Auth0 account.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/new-oauth2-server-config.png)
+
+::: note
+For the purposes of this example, we'll use the **Authorization Code grant type**, but you're free to use whichever grant type is most appropriate for your use case. Azure currently supports the following grant types: Authorization Code, Implicit, Resource Owner Password, Client Credentials.
+:::
+
+Provide values for the following fields:
+
+| Parameter | Description |
+| --------- | ----------- |
+| Name | a descriptive name for your authorization server |
+| Description | a description for your authorization server |
+| Client registration page URL | the page where users can create or manage their accounts; for the purposes of this example, we'll use `https://placeholder.contoso.com` as the placeholder |
+| Authorization code grant types | the grant type used for authorization |
+| Authorization endpoint URL | the URL Azure uses to make the authorization request |
+| Authorization request method | the HTTP method used by Azure to make the authorization request |
+| Token endpoint URL | the endpoint used to exchange authorization grants for access tokens; Auth0's can be reached at `https://auth0user.auth0.com/oauth/token` |
+| Client authentication methods | method used to authenticate the client; Auth0's is BASIC |
+| Access token sending method | the location of the access token in the sending method (typically the **Authorization header**) |
+| Default scope | specify a default scope if necessary |
+
+Because we're using the *authorization code** grant, we'll need to provide the Auth0 **client ID** and **client secret**.
```

Comment:
What is the client id and client secret? How do I get them from the dashboard? You say `the Auth0 client ID and client secret` while you mean the client id and client secret of the client you registered earlier in the dashboard.